src/theme/Footer/index.tsx

```diff
@@ -69,16 +79,29 @@ export const Footer = () => {
                 key={href}
                 underline='none'
               >
-                <ListItem>
+                <ListItem sx={{ py: 0 }}>
                   <ListItemText
                     primaryTypographyProps={{
-                      color: 'common.white',
+                      color: '#e3e3e3',
                       fontFamily: 'Lab Grotesque',
                       fontSize: 14,
                     }}
                   >
                     {label}
                   </ListItemText>
+                  {!isInternalUrl(href) && (
+                    <ListItemIcon
+                      sx={{
+                        minWidth: 'auto',
+                        ml: 1,
+                        '& svg': {
+                          color: 'common.white',
+                        }
+                      }}
+                    >
+                      <IconExternalLink />
```

Comment:
Thanks for adding this back in. Is it possible to get it back to the out-of-box look, as we had it before? 

Currently looks like this -- 
<img width="163" alt="Screen Shot 2022-12-01 at 2 06 07 PM" src="https://user-images.githubusercontent.com/56411016/205169189-7c1beb8e-c468-467e-a963-85d305349883.png">

This is how we'd like it (no gap) --
<img width="125" alt="Screen Shot 2022-12-01 at 2 04 11 PM" src="https://user-images.githubusercontent.com/56411016/205169005-0d37b2e2-c85a-4298-bcae-a3e3d18d59c0.png">

Reply:
How about show the icon on hover?

Reply:
We'd like to keep it static. We had it like Docusaurus has in their footer: https://docusaurus.io.